libvast/src/format/pcap.cpp

```diff
@@ -84,6 +86,11 @@ reader::reader(caf::atom_value id, const caf::settings& options,
     = community_id_ ? pcap_packet_type_community_id : pcap_packet_type;
   last_stats_ = {};
   discard_count_ = 0;
+  if (auto read_timeout_arg = caf::get_if<std::string>(&options, "import.read-"
+                                                                 "timeout")) {
+    if (auto read_timeout = to<vast::duration>(*read_timeout_arg))
```

Comment:
We should print an error if a read timeout was specified and cannot be parsed instead of silently ignoring it.

Reply:
Fair enough.